### PR TITLE
251 popular concepts

### LINKED ISSUE #251: Conceptschemes for popular concepts are hardcoded
The atramhasis/templates/subfooter-main.jinja2 template shows the most popular concepts or 4 conceptschemes.

Problems:
 * These are conceptschemes that are part of the development version. They will be different in every instance of Atramhasis
 * There's no guarantee that the instance will have 4 conceptschemes to show

Possible solutions:
 * Pick max. 4 conceptschemes at random?
 * Allow configuring the conceptschemes in the ini file?
 * Allow configuring and fallback to 4 random schemes when no configuration exists?
 * Make sure that the css works as well when there's only 1 or 2 conceptschemes

/cc @BartSaelen

### PR DESCRIPTION
**Pull request by taeymma**
fixes #251

### CODE REVIEW

File: atramhasis/templates/subfooter-main.jinja2 [hunk 1]
```diff
@@ -1,20 +1,41 @@
...
-  <div class="large-3 columns subfooter">
-    <h3>soorten bomen</h3>
-    {{ popular_concepts(request, 'SPECIES', 3) }}
-  </div>
-  <div class="large-3 columns subfooterright">
-    <h3>Materialen</h3>
-    {{ popular_concepts(request, 'MATERIALS', 3) }}
-  </div>
+  {% set topSchemes = request.registry.settings.get('layout.focus_conceptschemes') %}
+{#  {% set topSchemes = ['HERITAGETYPE', 'PERIOD', 'GEOGRAPHY', 'MATERIALS'] %}#}
```
Comment (koenedaele) [inline atramhasis/templates/subfooter-main.jinja2]: Can this be removed?

Comment (taeymma) [inline atramhasis/templates/subfooter-main.jinja2]: Removed example line.

**Comment by koenedaele** [comment]
I think `tests/conf_test.ini` needs to be updated.

**Comment by taeymma** [comment]
Added the layout.focus_conceptschemes and jinja2-filter to conf_test.ini

Build still fails.

**Comment by koenedaele** [comment]
Seems like the same error as in #249

**Pull request merged**